lua/astrocommunity/pack/clojure/init.lua

```diff
@@ -25,10 +34,28 @@ return {
       end
     end,
   },
+
+  -- Comment support `;;` and `;` (requires nvim 0.10.0)
+  -- first disable potential plugin clashes
+  { "numToStr/Comment.nvim", enabled = false },
+  { "JoosepAlviste/nvim-ts-context-commentstring", enabled = false },
+  -- Add ts-comment.nvim with Clojure comment strings
+  {
+    "folke/ts-comments.nvim",
+    filetype = { "clojure" },
+    opts = {
+      lang = {
+        clojure = { ";; %s", "; %s" },
+      },
+    },
+    event = "VeryLazy",
+    enabled = vim.fn.has "nvim-0.10.0" == 1,
+  },
+
```

Comment:
This should import the `ts-comment` entry from AstroCommunity and not just redefine it. Also only in neovim 0.10

Reply:
fixed in latest commit